tests/console/yast2_ntpclient.pm

```diff
@@ -34,15 +37,15 @@ sub run {
     assert_screen 'yast2_ntp-client_synchronize_without_daemon';
 
     # check Runtime Configuration Policy
-    send_key 'alt-r';
+    wait_screen_change { send_key 'alt-r'; };
     send_key 'up';
     assert_screen 'yast2_ntp-client_runtime_config_up';
     send_key 'down';
-    send_key 'ret';
+    wait_screen_change { send_key 'ret'; };
```

Comment:
Screen change is not recognized after down key press, as it's too small area. Is there any advantage of wrapping both send_key calls with wait_screen_change?